### PR TITLE
Feature: Added searchBar

### PR DESCRIPTION
**Pull request by aidensgithub**
## Added a simple SearchBar
Styled in Tailwind and the Search icon was taken from [Lucide](https://lucide.dev/icons/search)

**View on home screen**

image
---
**View on search results**

image
---
**View on no results**

image
---
**View on mobile**

image

### CODE REVIEW

File: src/ui/components/nav/components/SearchBar.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
+"use client";
```
Comment (mmiszy) [inline src/ui/components/nav/components/SearchBar.tsx:1]: I think this doesn't need to be a client component. You can use a server component with [server actions](https://nextjs.org/docs/app/api-reference/functions/server-actions) and call [`redirect`](https://nextjs.org/docs/app/api-reference/functions/redirect)

**Review by mmiszy — CHANGES_REQUESTED** [review]
Thank you so much for this contribution! I left a suggestion.

**Comment by zaiste** [comment]
@aidensgithub pretty cool addition!

File: src/ui/components/nav/components/SearchBar.tsx (hunk 1 above)
Comment (zaiste) [reply src/ui/components/nav/components/SearchBar.tsx]: @aidensgithub yes, it's more idiomatic in this context (App Router)

**Review by zaiste — DISMISSED** [review]

**Review by zaiste — APPROVED** [review]

**Pull request merged**